src/sources/aws_sqs/source.rs

```diff
@@ -143,6 +144,8 @@ impl SqsSource {
                         body.as_bytes(),
                         timestamp,
                         &batch,
+                        // TODO: Update this when updating the source to use log_namespaces.
+                        LogNamespace::Legacy,
```

Comment:
I need more ☕ I was reading this and was like "isn't that what you're suppose to be doing here???". Took me a while to read the filename properly 😆